trollmoves/dispatcher.py

```diff
@@ -39,6 +39,12 @@
         connection_uptime: 60
       filepattern: '{platform_name}_{start_time}.{format}'
       directory: /input_data/{sensor}
+      subscribe_addresses:
+        - tcp://127.0.0.1:40000
+      nameserver: 127.0.0.1
+      subscribe_services:
+        - service_name_1
+        - service_name_2
```

Comment:
Could these be documented as optional already here ?

Reply:
Done

Reply:
Done